Dockerfile

```diff
@@ -53,4 +50,5 @@ EXPOSE 26656
 # tendermint rpc
 EXPOSE 26657
 
-CMD ["/usr/bin/junod", "version"]
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/junod"]
```

Comment:
This will be a breaking change for consumers of this that assume the current run semantics, right? as in, they will have to change the command to omit the first part?

Reply:
Sense checking, as AFAICR all checks that use docker in CI rely on `exec` semantics, which will ignore any change to entrypoint

Reply:
> This will be a breaking change for consumers of this that assume the current run semantics, right? as in, they will have to change the command to omit the first part?

true. I realize that after checking CI

Reply:
Yeah, I think on that basis we have to maintain existing API just since folks will be relying on it in build scripts and all sorts of dev scenarios.

Reply:
> 

Let me revert it, since it can break existing folks running this

Reply:
thanks